lib/ecto/schema.ex

```diff
@@ -252,7 +252,7 @@ defmodule Ecto.Schema do
   defined so structs and changeset functionalities are available.
   """
 
-  @type t :: %{__struct__: atom}
```

Comment:
💁 this was the necessary one, the other one may be but wasn't in my project. I'm not sure the difference between these two things, they should be identical. Maybe it's in the `__meta__`?